ddtrace/tracer/tracer.go

```diff
@@ -171,9 +225,11 @@ func (t *tracer) worker() {
 			t.pushPayload(trace)
 
 		case <-ticker.C:
+			t.config.statsd.Incr("datadog.trace.flush.count", []string{"reason:scheduled"}, 1)
```

Comment:
`trace` -> `tracer` here and a few other spots.

Reply:
👍